packages/components/src/public/QueryModel/ExportMenu.tsx

```diff
@@ -106,13 +118,30 @@ const ExportMenuImpl: FC<ExportMenuImplProps> = memo(props => {
 
                         {exportOptions.map(option => (
                             <ExportMenuItem
-                                key={option.type}
+                                key={option.label}
                                 hasSelections={hasSelections}
                                 onExport={exportCallback}
                                 option={option}
                                 supportedTypes={supportedTypes}
                             />
                         ))}
+
```

Comment:
This is structured as such so that future devs can add multiple menu sections rather than just one. Potentially this is premature though.

Reply:
Seems fine to me.